docs/reference_guides/model_libraries/generic/costing/process_costing.rst

```diff
@@ -0,0 +1,865 @@
+Process Costing Library
+=======================
+
+.. contents:: Contents 
+    :depth: 4
+
+Introduction
+------------
+
+.. note:: The power plant costing method is available for most of the unit operations in the IDAES Unit Model Library (Compressor, CSTR, Flash, Heater, HeatExchanger, HeatExchangerNTU, PFR, PressureChanger, Pump, StoichiometricReactor, Turbine, and generic vessels).
+
+The process costing module outlines capital cost methodology for generic IDAES unit operations based on costing correlations that take unit configuration, material, size and other factors for unit costing into consideration.
+
+
+The framework includes factors for vessel shell thickness and dimensions, material properties for equipment shells and tubes, and tray types for distillation columns. The framework assumes a 2018 cost basis, and does not yet support heat exchangers requiring both shell and tube area (HX1D).
+
+When an instance of a unit model is passed to a `UnitModelCostingBlock`, a new sub-block is created 
```

Comment:
This common discussion should be moved to the discussion of the overall framework, and will cut down on space needed for the specific examples.